src/main/scala/viper/gobra/frontend/info/implementation/resolution/MemberResolution.scala

```diff
@@ -279,32 +279,43 @@ trait MemberResolution { this: TypeInfoImpl =>
         val methodLikeAttempt = tryMethodLikeLookup(typ, id)
         if (methodLikeAttempt.isDefined) methodLikeAttempt
         else typeSymbType(typ) match {
-          case pkg: ImportT => tryPackageLookup(pkg.decl, id)
+          case pkg: ImportT => tryPackageLookup(RegularImport(pkg.decl.importPath), id, pkg.decl)
           case _ => None
         }
     }
   }
 
   lazy val tryUnqualifiedPackageLookup: PIdnUse => Entity =
     attr[PIdnUse, Entity] {
-      id => {
-        val transitiveParent = transitiveClosure(id, tree.parent(_))
-        val entities = for {
-          // get enclosing PProgram for this PIdnUse node
-          program <- transitiveParent(id).collectFirst { case p: PProgram => p }
-          // consider all unqualified imports for this program (not package)
-          unqualifiedImports = program.imports.collect { case ui: PUnqualifiedImport => ui }
-          // perform a package lookup in each unqualifiedly imported package
-          results = unqualifiedImports.flatMap(ui => tryPackageLookup(ui, id))
-        } yield results
-        entities match {
-          case Some(Vector(elem)) => elem._1
-          case Some(v) if v.length > 1 => MultipleEntity()
-          case _ => UnknownEntity()
-        }
-      }
+      id =>
+        // Go is weird in the sense that it let's you redeclare built-in identifiers such as "error" but won't complain
+        // about the redeclaration. If the redeclaration happens in the same package, the redeclaration is
+        // used (instead of the built-in one). If the redeclaration happens in an imported package Go's behavior is not
+        // fully clear since "error" is not exported. However, we give higher precedence to the built-in one in Gobra
+        // to approximate Go's behavior.
+        tryUnqualifiedBuiltInPackageLookup(id).getOrElse(tryUnqualifiedRegularPackageLookup(id))
```

Comment:
What would happen if something is defined in the builtin package and a user defined pacakge? I'd guess that with the current setup the the user defined entity is overshadowed by builtin entity.

Reply:
There are two cases to distinguish:
- 1. you try to shadow a builtin member in the current package. You can find this case in `error-simple2.gobra`. Gobra and Go will use the redeclared member (Gobra uses it as this function is only invoked as a fallback if entity resolution fails)
- 2. you import a package that shadows a builtin member: as far as I can tell, all builtin members start with a lower case letter so they are ignored in Go. As Gobra does not care yet about an identifier's first letter, we give higher precendence to the builtin member (this is happening on this code line). Imho this mimics Go's behavior the best